aws-ia/taskcat
# Add Cloudformation waiter to delete operation

## Overview

Includes Cloudformation waiter for the delete operation. This is needed when we have a use-case to deploy sequential executions of Taskcat that leverages same set of resources as this could cause conflicts.
This is explained in this Issue: https://github.com/aws-ia/taskcat/issues/809

## Testing/Steps taken to ensure quality

Tested successfully Taskcat sequential deployment using this forked version

### Notes

Optional. Caveats, Alternatives, Other relevant information.

## Testing Instructions

 How to test this PR Start after checking out this branch (bulleted)
 * Include test case, and expected output
 * Test Taskcat sequential run with a Taskcat test clean in between.
